src/CertificateModule/controller/certificate.controller.ts

```diff
@@ -39,6 +39,18 @@ export class CertificateController {
     return this.mapper.toDtoList(await this.service.findAll());
   }
 
+  @Get('users/:userId')
```

Comment:
Não dá pra obter o id pelo token?

Reply:
Vms rever esse trecho. 
é bem provável que sim.

Reply:
Tudo que for referente a usuário, faz no UserController, exemplo: Certificados do usuário, ou cursos do usuário, essas coisas vão ficar no UserController, e na maioria das vezes vai ter um /me que pega o id pelo jwt e outro /:id que vai pegar o usuário pelo id passado na url

Reply:
Vamos fazer isso Raffa. 
Entendido Davi! 👊